frontend/assets/stylesheets/base/_type.scss

```diff
@@ -1,86 +0,0 @@
-/* ==========================================================================
-   Global typography settings
-   ========================================================================== */
-
-html {
```

Comment:
Split this file out, so element level styling goes in `_base.scss` (which was light anyway) and text helpers live in their own file.